warmup.py

```diff
@@ -240,17 +247,42 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if (self.rmax > (L / 2)):
+            raise ValueError('rmax > L/2')
+        else:
+            dist = np.zeros(sum(range(len(points))))                                                               #array of distance magnitudes between all point combinations
+            for i, point in enumerate(points):
+                neighbor_index = i + 1
+                point_shifted = point - (L * np.round(np.array(point) / L))                                        #wrap origin particle to periodic boundaries
+                while neighbor_index < len(points):
+                    neighbor = points[neighbor_index]                                                              #array of all other particles to compare to origin particle
+                    dist_vector = np.zeros_like(neighbor)                                                          #array of distances in x, y, z directions
+                    for j, (coord_neighbor, coord_point_shifted) in enumerate(zip(neighbor, point_shifted)):
+                        dist_vector[j] = coord_neighbor - coord_point_shifted - (L * np.round(coord_neighbor / L)) #wrap dist_vector to periodic boundaries
+                    dist[i] = np.linalg.norm(dist_vector)
+                    neighbor_index += 1
+            for i, bin_center in enumerate(self.centers):
```

Comment:
This type of search does work, but is very inefficient because it is a linear search for each pair distance. See my suggestion above for how to rewrite it. Also, I believe the way you are currently doing things overwrites the RDF each time accumulate is call, rather than adding a new sample to it. I forgot to add a unit test that covers this. :-)